orc8r/cloud/helm/orc8r/charts/nms/values.yaml

```diff
@@ -91,7 +91,7 @@ nginx:
     pullPolicy: Always
 
   service:
-    type: ClusterIP
+    type: LoadBalancer
```

Comment:
Hm. I assumed it already was a LB type... what am I missing?

Reply:
This branch has not been updated for a while so I will pull the master branch code on this.

Reply:
We have set this on AWS values template file but not by default.
https://github.com/magma/magma/blob/master/orc8r/cloud/deploy/terraform/orc8r-helm-aws/templates/orc8r-values.tpl#L204